.travis.yml

```diff
@@ -1,13 +1,14 @@
 language: php
 
 php:
-  - 5.5
-  - 5.6
   - 7.0
+  - 7.1
+  - 7.2
+  - nightly
 
 matrix:
   allow_failures:
-    - php: 7.0
+    - php: 7.1
```

Comment:
You say here that php 7.1 is allowed to fail. We are looking for the opposite, here all our developments must pass on PHP7.1 at least

We will also put versions 5.5, 5.6, 7.0 and nightly as allowed to fail.
The interest here is that we test on the public pipeline (it's a public repo) PHP versions that our customers could have BUT that we no longer want to support (http://php.net/supported -versions.php to see PHP versions that are no longer maintained) and if tests ever pass they can use the package, charge them to see that they are on an outdated system.
On the other hand, for the nightly, we want to anticipate the future update of PHP (nightly -> nightly build of the last version of PHP (potentially php3 if I say no stupidity)), so we are in the case we want.

```
matrix:
  allow_failures:
    - php: 5.5
    - php: 5.6
    - php: 7.0
    - php: nightly
```